### DISCUSSION TITLE
React to a room invitation ("handle" the WhatsApp bridge)

### DISCUSSION THREAD

**Discussion by jacotec** [Skylar-Tech/node-red-contrib-matrix-chat#69] [Q&A] [answered]
This plugin looks really interesting and I'm questioning if I can use it to add missing functionality to the WhatsApp bridge relay functionality.

The basic question is: Can the plugin react to an invitation to a room?

What I need is basically:

- I have a user "relay" which is connected to the WhatsApp bridge. But I want to run this "unmanned".
- When a new person writes a message to WhatsApp, the bridge creates a new room and invites the user "relay".
- The node red should "see" the invite and:
- accept the invitation
- send a message "!wa set-relay" into the new room
- invite two other users to this room

Will this be possible?

Edit: I've played a bit with it ... I can simply realize all I need, but I don't get any message via the receive node when the user is invited to a new room. So I can't trigger a sequence when entering a new room. Any workaround here?

**Comment by skylord123** [comment, marked as answer]
We added a new node in the latest release for this. I also replied to your issue here about it: https://github.com/Skylar-Tech/node-red-contrib-matrix-chat/issues/70

Thanks!
